src/UiAutocomplete.vue

```diff
@@ -59,7 +59,7 @@
                         ref="suggestions"
 
                         :highlighted="highlightedIndex === index"
-                        :key="index"
+                        :key="JSON.stringify(suggestion)"
```

Comment:
This could potentially lead to an error if `suggestion` has a circular reference.

Probably not likely, but it's something to keep in mind.

Maybe we could just require a unique `key` property on the suggestion if it's an object?

Reply:
Not sure how likely there is a good use case for a circular reference within an autocomplete though?

At that level it seems more like a coding error.

Could require a key, but automating is nice. Tbf, I'd probably slice it too after stringify.